torch/csrc/jit/script/init.cpp

```diff
@@ -392,7 +392,15 @@ FunctionSchema getSchemaWithNameAndDefaults(
     auto it = default_args.find(arg.name());
     if (it != default_args.end()) {
       try {
-        IValue value = toIValue(it->second, arg.type());
+        IValue value;
+        auto n = arg.N();
+        auto list_type = arg.type()->cast<ListType>();
+        if (n && *n > 0 && list_type) {
```

Comment:
Nit: could move the arg.type()->cast<ListType>() cast into the conditional, and could have only one call to toIValue